backends/openweathermap.org.go

```diff
@@ -55,6 +57,15 @@ type dataBlock struct {
 	} `json:"rain"`
 }
 
+type openWeatherErrorReponse struct {
+	Cod     any    `json:"cod"`
```

Comment:
Lets call this `Code` internally, even if the API spec we parse is just `cod`.

Reply:
I have changed Cod to Code everywhere for ```openWeatherErrorResponse```